### DISCUSSION TITLE
O que fazer de Portefólio?

### DISCUSSION THREAD

**Discussion by AdemilsonSimiao** [frontendbr/forum#2251] [Discussão]
Gostaria de saber o que fazer de portefólio?

**Comment by theandersonn** [comment]
Sua dúvida é sobre tecnologia para criar um portfólio ou o conteúdo?

Recentemente conheci esse book aqui, gostei. Recomendo a leitura e inclusive aborda esses dois aspectos que citei.
https://www.joshwcomeau.com/effective-portfolio/
